src/components/task-card/components/header/Header.tsx

```diff
@@ -0,0 +1,12 @@
+import classes from './styles.module.css'
+
+export default function Header() {
```

Comment:
make the component flexible, the consumer should be able to change the Order and Text of the heading